tests/data/packages/requiresPaste/pyproject.toml

```diff
@@ -0,0 +1,9 @@
+[build-system]
+requires = ['flit_core >=2,<4']
+build-backend = 'flit_core.buildapi'
+
+[tool.flit.metadata]
+module = 'requiresPaste'
+author = 'A. Random Developer'
```

Comment:
```suggestion
author = 'A Random Developer'
```

Reply:
This is actually taken from [the packaging guide from python.org](https://packaging.python.org/guides/distributing-packages-using-setuptools/#author).  I think they intended to make `A.` looks like initial of a real person name :smile:

Reply:
Hmm... this isn't as funny now as it was in my head when I suggested the change. :)

Reply:
Not entirely related but this reminds me of [SICP's characters](https://en.wikipedia.org/wiki/Structure_and_Interpretation_of_Computer_Programs#Characters).